src/cpp/CMakeLists.txt

```diff
@@ -20,9 +20,10 @@ set_sources(
     Cdr.cpp
     FastCdr.cpp
     FastBuffer.cpp
+    exceptions/BadOptionalAccessException.cpp
+    exceptions/BadParamException.cpp
     exceptions/Exception.cpp
     exceptions/NotEnoughMemoryException.cpp
-    exceptions/BadParamException.cpp
     FastCdr.rc
```

Comment:
Is this file still required or can it be removed with the other Make files?

Reply:
It was moved. I sorted by name.

Reply:
It is used on windows library.